pkgs/development/python-modules/urllib3/default.nix

```diff
@@ -22,11 +23,12 @@ buildPythonPackage rec {
 
   doCheck = false;
 
-  buildInputs = [ coverage tornado mock nose psutil pysocks ];
+  buildInputs = [ pytest mock tornado ];
+  propagatedBuildInputs = [ pyopenssl cryptography idna certifi ipaddress pysocks ];
```

Comment:
Archlinux seems to have banned `certifi` from their core repos as it introduces a new ssl trust store.
(But this is just an unrelated issue)

Reply:
That's indeed a bigger issue. Would be nice to see a RFC on how we are going to handle certificates. I think some choices were made but they're not documented.

Reply:
cc @fpletz @grahamc